plugins/hu.elte.txtuml.api.model/src/hu/elte/txtuml/api/model/ModelExecutor.java

```diff
@@ -66,6 +60,7 @@
 	 * an uninstantiatable class.
 	 */
 	private ModelExecutor() {
+		thread.start();
```

Comment:
This code never actually runs.

See the JavaDoc:
"Sole constructor of <code>ModelExecutor</code>, which is designed to be
an ***uninstantiatable*** class."